index.js

```diff
@@ -13,18 +11,6 @@ var runner = new Runner({
 	match: opts.match
 });
 
-// check if the test is being run without AVA cli
-var isForked = typeof process.send === 'function';
-
-if (!isForked) {
-	var fp = path.relative('.', process.argv[1]);
-
-	console.log();
-	console.error('Test files must be run with the AVA CLI:\n\n    ' + chalk.grey.dim('$') + ' ' + chalk.cyan('ava ' + fp) + '\n');
-
-	process.exit(1); // eslint-disable-line
-}
```

Comment:
it's not in `wrap-send`, it's in `send-to-parent` (it's own module - as requested), If there isn't a parent, it complains.

It seems entirely sensible that `require('send-to-parent')` would throw if you were no parent.